.github/workflows/testBuildAndPublish.yml

```diff
@@ -0,0 +1,143 @@
+# GitHub Actions Workflow created for testing and publishing the plugin in following steps:
+# - validate Gradle Wrapper,
+# - run test and verifyPlugin tasks,
+# - run buildPlugin task and prepare artifact for the further tests,
+# - run IntelliJ Plugin Verifier,
+# - publish plugin if the workflow was triggered by a release
+
+# Docs:
+# - GitHub Actions: https://help.github.com/en/actions
+# - IntelliJ Plugin Verifier GitHub Action: https://github.com/ChrisCarini/intellij-platform-plugin-verifier-action
+
+on:
+  push:
+    branches:
+      - '**'
+  pull_request:
+  release:
+    types: [released]
+
+name: Test, Build & Publish Plugin
+
+jobs:
```

Comment:
I basically copied the jobs from `build.yml` & `release.yml` into this one workflow. (And fixed the seemingly broken release job 🙈 ). It's hard to see here because I renamed the file :(